mtp_noms_ops/apps/security/views.py

```diff
@@ -291,3 +279,21 @@ def prisoner_image_view(request, prisoner_number):
         return HttpResponseRedirect(static('images/placeholder-image@2x.png'))
     else:
         return HttpResponseRedirect(static('images/placeholder-image.png'))
+
+
+def prisoner_details_view(request, prisoner_number):
+    response_data = {}
+    if nomis_api_available(request) and prisoner_number:
+        try:
+            location = get_location(prisoner_number)
+            if 'housing_location' in location:
+                levels = {
+                    level['type']: level['value'] for level in
+                    location['housing_location']['levels']
+                }
+                response_data['housing_location'] = '-'.join([
+                    levels['Wing'], levels['Landing'], levels['Cell']
+                ])
+        except RequestException:
+            logger.warning('Could not load location for %s' % prisoner_number)
+    return JsonResponse(response_data)
```

Comment:
how do you feel about some caching? like with photos: `patch_cache_control(response, private=True, max_age=???)`